interactors/rotate_block_interactor.py

```diff
@@ -0,0 +1,9 @@
+from entities.gameboard_entity import Gameboard
+from entities.block_entity import Block
+
+class RotateBlockInteractor():
+    def __init__(self, gameboard: Gameboard):
+        self.gameboard = gameboard
+
+    def rotate_block(self, block: Block):
+        print('rotate')
```

Comment:
This print statement will be replaced later on with appropriate logic?

Reply:
Yup, it is only a placeholder